apps/i3wm/i3wm.talon

```diff
@@ -61,13 +61,13 @@ vertical (shell|terminal):
 
 # XXX - just replace with shuffle eventually?
 # XXX - like also need to match the generic talon commands
-(shuffle|move (win|window) [to] port) <number_small>:  user.system_command("i3-msg move container to workspace number {number_small}")
-(shuffle|move (win|window) [to] port ten): user.system_command("i3-msg move container to workspace number 10")
-(shuffle|move (win|window) [to] last port): user.system_command("i3-msg move container to workspace back_and_forth")
-(shuffle|move (win|window) left): user.system_command("i3-msg move left")
-(shuffle|move (win|window) right): user.system_command("i3-msg move right")
-(shuffle|move (win|window) up): user.system_command("i3-msg move up")
-(shuffle|move (win|window) down): user.system_command("i3-msg move down")
+(shuffle|move) [(win|window)] [to] port <number_small>:  user.system_command("i3-msg move container to workspace number {number_small}")
+(shuffle|move) [(win|window)] [to] last port: user.system_command("i3-msg move container to workspace back_and_forth")
+(shuffle|move) flipper: user.system_command("i3-msg move container to workspace back_and_forth")
```

Comment:
is this distinct from `port flip | flipper`?

Reply:
`port flip | flipper` moves an entire workspace (think virtual desktop) between outputs (monitors)
this is moving a window between workspaces